docker-compose.override.yml

```diff
@@ -209,7 +209,7 @@ services:
         environment:
             PUBLIC_URL: "/network-explorer"
             REACT_APP_STREAMR_API_URL: "${STREAMR_BASE_URL}/api/v1"
-            REACT_APP_TRACKER_REGISTRY_PROVIDER: "http://10.200.10.1:8545"
+            REACT_APP_TRACKER_REGISTRY_PROVIDER: "${STREAMR_BASE_URL}/mainchain-rpc/http/"
             REACT_APP_TRACKER_REGISTRY_ADDRESS: "0xBFCF120a8fD17670536f1B27D9737B775b2FD4CF"
             REACT_APP_MAPBOX_TOKEN: "pk.eyJ1IjoibWF0dGlubmVzIiwiYSI6ImNrNWhrN2FubDA0cGgzam1ycHV6Nmg2dHoifQ.HC5_Wu1R-OqRLza1u6P3Ig"
```

Comment:
Is this secret? Well, since this is a frontend app, it's going to leak anyway. So probably not much that can be done there. Otherwise LGTM